src/pages/OrgRegister.tsx

```diff
@@ -18,7 +18,7 @@ export default function OrgRegister() {
             </label>
             <input
               type="text"
-              className="border border-primary w-full p-3 dark:border-white dark:bg-[#1F2A37] rounded mb-4"
+              className="border text-[13px] border-primary w-full p-3 dark:border-white dark:bg-[#1F2A37] rounded mb-4"
```

Comment:
On dark mode, the input text is not configured to match the color used in the design and is defaulting to black. Refer to this picture

![image](https://user-images.githubusercontent.com/28519395/178074060-6dd5f87b-1153-48be-8a24-04632a8ccd1f.png)